spcs-basic/build-and-push.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+DB_NAME="spcs_na_db"
+SCHEMA_NAME="public"
+IMAGE_REPO_NAME="images"
+SERVICE_NAME="spcs_na_service"
```

Comment:
This is really more of an `IMAGE_NAME`. Service name is going to be the name in Snowflake of the instantiated service based on this image.